sdk/lib/rtl/actctx.c

```diff
@@ -2989,7 +2998,26 @@ static NTSTATUS open_nt_file( HANDLE *handle, UNICODE_STRING *name )
     attr.ObjectName = name;
     attr.SecurityDescriptor = NULL;
     attr.SecurityQualityOfService = NULL;
-    return NtOpenFile( handle, GENERIC_READ | SYNCHRONIZE, &attr, &io, FILE_SHARE_READ, FILE_SYNCHRONOUS_IO_ALERT );
+    return NtOpenFile( handle, GENERIC_READ | SYNCHRONIZE, &attr, &io,
+                       FILE_SHARE_READ | FILE_SHARE_DELETE, FILE_SYNCHRONOUS_IO_ALERT );
+}
+
+static NTSTATUS find_first_manifest_resource_in_module( HANDLE hModule, const WCHAR **resname )
```

Comment:
This seems fishy, iirc windows does not use just *any* manifest, but the numbers actually have meaning.

Reply:
CREATEPROCESS_MANIFEST_RESOURCE_ID is 1 so it should be found first. In practice I don't think anyone has more than one manifest? Does ROS support ISOLATION_AWARE_ENABLED at all?

Reply:
> CREATEPROCESS_MANIFEST_RESOURCE_ID is 1 so it should be found first. In practice I don't think anyone has more than one manifest? Does ROS support ISOLATION_AWARE_ENABLED at all?

ReactOS does partially support this, but there is `1`, `2` and `123` that I know of,
just selecting the 'first' one might actually use a manifest when it shouldn't be used.